### ISSUE TITLE
Unable to configure acronyms: undefined method `new' for Strings::Case:Module

### ISSUE THREAD

**Issue by baka-san** [piotrmurach/strings-case#4]
Unable to configure acronyms. Using Ruby 2.7.4 and Rails 6.1.4.1. 

### Setup
```ruby
# config/initializers/inflections.rb
strings = Strings::Case.new
strings.configure do |config|
  config.acronym "HTTP"
  config.acronym "XML"

  # or config.acronym "HTTP", "XML"
end
```

### Error
```
/my_project/config/initializers/inflections.rb:33:in `<main>': undefined method `new' for Strings::Case:Module (NoMethodError)
```

**Comment by piotrmurach** [comment]
Hi Grant,

Thank you for giving `strings-case` a try. 

You're seeing the errors because the readme documentation refers to what's in the current branch. I change documentation as I write the functionality. I also update the [changelog](https://github.com/piotrmurach/strings-case/blob/master/CHANGELOG.md) if you're curious to know what's changed.

Please use the dropdown to select the `v0.3.0` tag to read documentation for the released version.

I'd like to hear your thoughts on the unreleased version if you have the time to test it. Unfortunately, I cannot promise you when the new version will be released.

**Comment by baka-san** [comment]
@piotrmurach 

Sorry for the slow reply. I've included `v0.3.0` directly and the issue is gone and everything seems to be working fine so far. Thank you!

**Comment by piotrmurach** [comment]
@baka-san 大丈夫。It's a good prompt for me to wrap a new release shortly. Stay tuned.

**Comment by piotrmurach** [comment]
@baka-san This is released now as `v0.4.0`.
